tests/multiple-tests-with-exception/expected_results.json

```diff
@@ -2,22 +2,26 @@
   "status" : "fail",
   "tests" : [
     {
-      "name" : "MultipleWithExceptionTests.testAdd",
-      "status" : "pass"
+      "name" : "testAdd",
+      "status" : "pass",
+      "test_code" : "XCTAssertEqual(sum(2, 3), 5, \"2+3 should equal 5\")"
     },
     {
-      "name" : "MultipleWithExceptionTests.testSub",
-      "status" : "pass"
+      "name" : "testSub",
+      "status" : "pass",
+      "test_code" : "XCTAssertEqual(sub(2, 3), -1)"
     },
     {
-      "name" : "MultipleWithExceptionTests.testMul",
-      "status" : "pass"
+      "name" : "testMul",
+      "status" : "pass",
+      "test_code" : "XCTAssertEqual(mul(3, 2), 6)"
     },
     {
-      "message" : "/solution/Tests/MultipleWithExceptionTests/MultipleWithExceptionTests.swift:19: error: MultipleWithExceptionTests.testThrow : XCTAssertEqual threw error \"testError(\"Oh noes! Div by zeroes!!!\")\" - ",
-      "name" : "MultipleWithExceptionTests.testThrow",
-      "status" : "fail"
+      "message" : "/solution/Tests/MultipleWithExceptionTests/MultipleWithExceptionTests.swift:19: error: MultipleWithExceptionTests.testThrow : XCTAssertEqual threw error \"testError(\"Oh noes! Div by zeroes!!!\")\" -",
+      "name" : "testThrow",
+      "status" : "error",
```

Comment:
This should be a `fail`, as `error` is for when the tests couldn't even run (which they did in this case).